keras_cv/layers/object_detection_3d/voxel_utils.py

```diff
@@ -118,20 +99,15 @@ def compute_voxel_origin(
       voxel_origin: [dim] the voxel origin.
     """
     voxel_origin = spatial_size[::2]
-    voxel_origin = tf.constant(
-        [o / v for o, v in zip(voxel_origin, voxel_size)], dtype=tf.float32
+    voxel_origin = np.array(
+        [o / v for o, v in zip(voxel_origin, voxel_size)], "float32"
     )
-    voxel_origin = tf.math.round(voxel_origin)
-    voxel_origin = tf.cast(voxel_origin, dtype=tf.int32)
+    voxel_origin = np.round(voxel_origin)
+    # voxel_origin = np.cast(voxel_origin, dtype="int32")
```

Comment:
why did this get commented out? should we delete forever, or still a todo here?

Reply:
Ahh yeah we can get rid of this